clang/test/SemaSYCL/deferred-diagnostics-emit.cpp

```diff
@@ -16,6 +16,12 @@ class type_info;
 typedef __typeof__(sizeof(int)) size_t;
 } // namespace std
 
+template<typename T>
+void Dep() {
+	enum Mode { N, E, O, EO };
+	Mode m = N;
+}
```

Comment:
Is that related to the changes?

Reply:
Oh, it can't be observed with the current version. That's guarding one of the new checks, one asserts if `Ty` is a dependant type, IIRC `Ty->isSpecificBuiltinType()`. The assert is triggered with the GCC 13 headers.